### PR TITLE
Follow, Unfollow user

### PR DESCRIPTION
**Pull request by vinhnd2010**

### CODE REVIEW

File: app/controllers/relationships_controller.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class RelationshipsController < ApplicationController
+  before_action :logged_in_user
+
+  def create
+    @user = User.find params[:followed_id]
+    current_user.follow @user
+    respond_to do |format|
+      format.html { redirect_to @user }
```
Comment (trungnn142) [inline app/controllers/relationships_controller.rb]: space in {}

File: app/controllers/relationships_controller.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      format.js
+    end
+  end
+
+  def destroy
+    @user = Relationship.find(params[:id]).followed
+    current_user.unfollow @user
+    respond_to do |format|
+      format.html { redirect_to @user }
```
Comment (trungnn142) [inline app/controllers/relationships_controller.rb]: space in {}

File: app/views/relationships/create.js.erb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+$("#follow_form").html("<%= escape_javascript(render('users/unfollow')) %>");
+$("#followers").html('<%= @user.followers.count %>');
```
Comment (trungnn142) [inline app/views/relationships/create.js.erb]: replace ' => "

File: app/views/relationships/destroy.js.erb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+$("#follow_form").html("<%= escape_javascript(render('users/follow')) %>");
+$("#followers").html('<%= @user.followers.count %>');
```
Comment (trungnn142) [inline app/views/relationships/destroy.js.erb]: replace '=>"

File: app/views/users/_user.html.erb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<li>
+  <section class="user_list">
+    <%= link_to user.name, user %>
+  </section>
+  <% if current_user.admin? && !current_user?(user) %>
+    | <%= link_to "delete", user, method: :delete,
+                  data: {confirm: "You sure?"} %>
```
Comment (ThuBM) [inline app/views/users/_user.html.erb]: indent

File: app/controllers/users_controller.rb [hunk 6]
```diff
@@ -25,6 +25,20 @@ def update
     end
   end
 
+  def following
```
Comment (dieunb) [inline app/controllers/users_controller.rb]: RestFull

File: app/controllers/users_controller.rb [hunk 7]
```diff
@@ -25,6 +25,20 @@ def update
...
+    @title = t "user.following"
+    @user = User.find params[:id]
+    @users = @user.following.paginate(page: params[:page])
+    render "show_follow"
+  end
+
+  def followers
```
Comment (dieunb) [inline app/controllers/users_controller.rb]: RestFul

**Comment by ThuBM** [comment]
OK

**Pull request merged**